src/unistd.rs

```diff
@@ -751,8 +767,10 @@ impl Whence {
             &Whence::SeekSet => libc::SEEK_SET,
             &Whence::SeekCur => libc::SEEK_CUR,
             &Whence::SeekEnd => libc::SEEK_END,
-            &Whence::SeekData => 3,
-            &Whence::SeekHole => 4
+            #[cfg(any(target_os = "linux", target_os = "dragonfly", target_os = "freebsd"))]
```

Comment:
Don't do this here, delete this whole `impl` and just assign values within the enum like `SeekSet = libc::SEEK_SET,`. You'll also want to say that it's a `i32` datatype. Look at how it's implemented for [`Signal`](https://github.com/nix-rust/nix/blob/master/src/sys/signal.rs#L19).